gradle/libs.versions.toml

```diff
@@ -218,7 +218,7 @@ wiremockJunit5 = { module = "ru.lanwen.wiremock:wiremock-junit5", version.ref =
 # plugin dependencies
 plugin-errorprone = { module = "net.ltgt.errorprone:net.ltgt.errorprone.gradle.plugin", version = "4.2.0" }
 plugin-license = { module = "gradle.plugin.com.hierynomus.gradle.plugins:license-gradle-plugin", version = "0.16.1" }
-plugin-nebulaRelease = { module = "com.netflix.nebula:nebula-release-plugin", version = "18.0.8" }
+plugin-nebulaRelease = { module = "com.netflix.nebula:nebula-release-plugin", version = "20.2.0" }
```

Comment:
Since this didn't end up fixing anything, we can revert it in this PR and look to upgrade these plugins separately, since dependabot doesn't do it for us for new major versions.